after/syntax/zsh.vim

```diff
@@ -1,122 +1,169 @@
+            \ contains=ZinitIceSingleQuoteParam
+syn region ZinitIceWithParam matchgroup=ZinitIce start=+\s\%(wrap\|ps-on-unload\|ps-on-update\)'+ skip=+\\'+ end=+'+ skipwhite contained
+
+" zinit packages
+syn region ZinitIceWithParam matchgroup=ZinitIce start=+\s\%(param\)'+ skip=+\\'+ end=+'+ skipwhite contained
+            \ nextgroup=@ZinitLine,ZinitContinue
+            \ contains=ZinitIceSingleQuoteParam
+
+" added by the existing annexes
+syn region ZinitIceWithParam matchgroup=ZinitIce start=+\s\%(fbin\|sbin\|gem\|node\|pip\|fmod\|fsrc\|ferc\)'+ skip=+\\'+ end=+'+ skipwhite contained
+            \ nextgroup=@ZinitLine,ZinitContinue
+            \ contains=ZinitIceSingleQuoteParam
+syn region ZinitIceWithParam matchgroup=ZinitIce start=+\s\%(dl\|patch\|submods\|cargo\|dlink\|dlink0\)'+ skip=+\\'+ end=+'+ skipwhite contained
+            \ nextgroup=@ZinitLine,ZinitContinue
+            \ contains=ZinitIceSingleQuoteParam
+
+syn match ZinitIceSingleQuoteParam +[^']*+ contained
+
+" ices that doens't take a param
+syn match ZinitIce '\s\%(teleid\|svn\)\>'ms=s+1 skipwhite contained
+            \ nextgroup=@ZinitLine,ZinitContinue
+syn match ZinitIce '\s\%(wait\|cloneonly\)\>'ms=s+1 skipwhite contained
+            \ nextgroup=@ZinitLine,ZinitContinue
+syn match ZinitIce '\s\%(silent\|lucid\|notify\)\>'ms=s+1 skipwhite contained
+            \ nextgroup=@ZinitLine,ZinitContinue
+syn match ZinitIce '\s\%(blockf\|nocompletions\)\>'ms=s+1 skipwhite contained
+            \ nextgroup=@ZinitLine,ZinitContinue
+syn match ZinitIce '\s\%(run-atpull\|nocd\|make\|countdown\|reset\)\>'ms=s+1 skipwhite contained
+            \ nextgroup=@ZinitLine,ZinitContinue
+syn match ZinitIce '\s!\?\%(sh\|bash\|ksh\|csh\)\>'ms=s+1 skipwhite contained
+            \ nextgroup=@ZinitLine,ZinitContinue
+syn match ZinitIce '\s\%(id-as\|nocompile\|reset-prompt\|trackbinds\|aliases\|light-mode\)\>'ms=s+1 skipwhite contained
+            \ nextgroup=@ZinitLine,ZinitContinue
+syn match ZinitIce '\s\%(is-snippet\)\>'ms=s+1 skipwhite contained
+            \ nextgroup=@ZinitLine,ZinitContinue
+
+" ices that doens't take a param, from zinit packages
+syn match ZinitIce '\s\%(pack\|git\|null\)\>'ms=s+1 skipwhite contained
+            \ nextgroup=@ZinitLine,ZinitContinue
+
+" ices that doens't take a param, added by the existing annexes
+syn match ZinitIce '\s\%(notest\|rustup\|default-ice\|skip\|debug\)\>'ms=s+1 skipwhite contained
+            \ nextgroup=@ZinitLine,ZinitContinue
+
+" additional Zsh and zinit functions
+syn match ZshAndZinitFunctions '\<\%(compdef\|compinit\|zpcdreplay\|zpcdclear\|zpcompinit\|zpcompdef\)\>'
+
+" highlights
+hi def link ZinitCommand             Statement
+hi def link ZinitCommand             Title
+hi def link ZinitIceCommand          Title
+hi def link ZinitPluginCommand       Title
+hi def link ZinitSnippetCommand      Title
+hi def link ZinitForCommand          zshRepeat
+" hi def link ZinitContinue            zshOperator
```

Comment:
Should this be commented out?

Reply:
It's commented out since it sort of hijacks any normal line continuation - but only inside a zinit 'block', line continuations aren't recognized by the zsh syntax file.

I left it in so that all groups are present. But it really shouldn't link to zshOperator in the comment, if it should stay at all. I guess linking it to Normal should work.